nss-day-cohort-39/custom-theme-maker-c-sharp
# added event listeners to change background colors of all components

### Changes

- Added event listeners to change background colors of Scores, News, and Favorites Lists to red/purple/blue/green

### Testing

- Make sure all color buttons change backgrounds of all modules
- Make sure buttons change to new colors
